install_config/topics/ocp_osp_networks.adoc

```diff
@@ -0,0 +1,48 @@
+==== Creation of {rhocp} Networks via OpenStack
+
+When deploying {rhocp} on {rhosp} as described in this segment,
+the requirements are two networks -- _public_ and _internal_ network.
+
+===== Public Network
+
+The _public_ network is a network that contains external access and can be reached
+by the outside world. The  _public_ network creation can be only done by an
+OpenStack administrator.
+
+The following commands provide an example of creating an OpenStack provider
+network for _public_ network access.
+
+As an OpenStack administrator,
+
+[subs=+quotes]
+----
+$ source /path/to/examplerc
+
+$ openstack network create *<public-net-name>* \
+  --external \
+  --provider-network-type flat \
+  --provider-physical-network datacentre
+
```

Comment:
it is better to add “check with command: openstack network list”

Reply:
Failed to execute @rlopez133 
# openstack subnet create public_sub_network \
> --network public_network \
> --dhcp \
> --allocation-pool start=192.168.122.30,end=192.168.122.200 \
> --gateway 192.168.122.1 \
> --subnet-range 192.168.122.0/24

HttpException: Unknown error

Reply:
This will fail to execute because you need to be an OSP administrator otherwise you aren't allowed to create openstack provider networks.